dashboard/app/models/script.rb

```diff
@@ -1472,6 +1472,8 @@ def get_assessment_script_levels
   def get_feedback_for_section(section)
     feedback = {}
     script_levels.each do |script_level|
+      # Filter down to only the levels a teacher can give feedback on
+      next unless ["Applab", "Gamelab", "Weblab"].include?(script_level.level.type)
```

Comment:
Instead of hard-coded level types here, let's use the existing level polymorphism so we can ask
```rb
next unless script_level.level.can_have_feedback?
```